src/core/mainloop/fooz

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This file appears to have been mistakenly committed.

Reply:
Removed in 8df602ebb702b1